src/useRaf/useRaf.ts

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { useIsomorphicLayoutEffect } from '..';
+
+/**
+ *
+ * @param ms milliseconds for how long to keep rendering component
+ * @param delay delay in milliseconds after which to start re-rendering component
+ * @returns number
+ */
+export function useRaf(ms = 1e12, delay = 0): number {
```

Comment:
1. This hook should have handles to stop and start re-rendenring process, start should apply optional rerender duration.
2. In implementation it should reuse `useRafCallback` hook, that is already implemented in this package.
3. `elapsed` should use `useSafeState`, since it is updated asynchronously - there is a chance, that component will be unmounted before state update.